Resources/Locale/en-US/gas-analyzer.ftl

```diff
@@ -0,0 +1,12 @@
+### UI
+
+# Item status of the gas analyzer. {$status} indicates how dangerous the surrounding area is
+comp-gas-analyzer-pressure-status = Pressure: [color={$color}]{$status}[/color]
+
+# Gas analyzer menu current zone status information
+gas-analyzer-error = Error: {$error}
+gas-analyzer-pressure = Pressure: { PRESSURE($pressure, "0.##") }
```

Comment:
PRESSURE doesn't take a format string.
It's just `{ PRESSURE($pressure) }`

Reply:
oh damn, i was testing and forgot to remove that